jerry-core/include/jerryscript-debugger.h

```diff
@@ -28,12 +28,22 @@ extern "C"
  */
 
 /**
+ * Types for the client source wait and run method.
+ */
+typedef enum
+{
+  JERRY_DEBUGGER_WAIT_AND_RUN_FAIL = 0, /**< error occured while receive */
+  JERRY_DEBUGGER_WAIT_AND_RUN_SUCCESS = 1, /**< the source has been received */
```

Comment:
Come to think of it, I would remove the WAIT_AND_RUN since it extends the string too much. What about:

JERRY_DEBUGGER_SOURCE_RECEIVE_FAILED 
JERRY_DEBUGGER_SOURCE_RECEIVED